finance-api/api-reference.md

```diff
@@ -0,0 +1,333 @@
+|403 Forbidden |Returned when the scope being used in the request is incorrect.  
+</details>
+
+<details open="true">
+<summary>Click to hide response attributes.</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the upcoming payout. 
+|totalBalance |integer |The account balance in the currency's smallest unit. For example, 300 with currency GBP is £3. It can be negative, such as when refunds are greater than sales.
+|currencyId |string |The currency of the account. For example, `GBP`. 
+|nextPayoutAmount |integer |The amount of money to be paid out to the merchant.  
+|discountRemaining |integer |The amount of discounts that remains in merchant's vouchers. The vouchers are offered by Zettle.<br/>For example, a merchant has a voucher worthy £100 from a Zettle marketing campaign. For a transaction of £200, Zettle will subtract £2 for the transaction fee from the voucher. Then the merchant will have a remaining discount of £98. 
+|periodicity |string |The period between each payout that is set by the merchant. It can be `DAILY`, `WEEKLY` or `MONTHLY`.   
+</details>
+
+
+## Examples
+
+### Fetch current balance for a liquid account
+In the following example, as `at` is not specified, the current balance £1.95 is fetched by default.
+
+Request
+```
+GET /organizations/self/accounts/liquid/balance
+```
+Response
+```json
+{
+    "data": {
+        "totalBalance": 195,
+        "currencyId": "GBP"
+    }
+}
+```
+
+### Fetch transactions for a liquid account
+The following example fetches transactions from the merchant's liquid account from 1 January, 2020 to 31 December, 2020.
+
+Request
+```
+GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-12-31
+```
+
+Response
+```json
+{
+    "data": [
+        ...
+        {
+            "timestamp": "2020-07-04T20:16:44.309+0000",
+            "amount": 381,
+            "originatorTransactionType": "CARD_PAYMENT_FEE_REFUND",
+            "originatingTransactionUuid": "30cef6e2-be09-11ea-a8e4-bce028663c34"
+        },
+        {
+            "timestamp": "2020-07-04T20:16:44.309+0000",
+            "amount": -20610,
+            "originatorTransactionType": "CARD_REFUND",
+            "originatingTransactionUuid": "30cef6e2-be09-11ea-a8e4-bce028663c34"
+        },
+        ...
+        {
+            "timestamp": "2020-06-27T23:52:18.327+0000",
+            "amount": 649,
+            "originatorTransactionType": "PAYMENT_FEE", // In case of refund
+            "originatingTransactionUuid": "690c99ea-b6ef-11ea-9730-7ef7aeff642d"
+        },
+        {
+            "timestamp": "2020-06-27T23:52:18.327+0000",
+            "amount": -35100,
+            "originatorTransactionType": "PAYMENT", // In case of refund
+            "originatingTransactionUuid": "690c99ea-b6ef-11ea-9730-7ef7aeff642d"
```

Comment:
I liked @iznatalia 's idea of putting the comment in the amount, not in the type. The amount is negative because it's a refund.